### PR TITLE
Notif ui

### PR DESCRIPTION
**Pull request by Astalaseven**
- [x] Switch to RecyclerView
- [ ] Adapt UI to be more readable

### CODE REVIEW

File: app/src/main/java/gitlin/kothub/ui/NotificationActivity.kt [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
 import gitlin.kothub.github.api.data.Notifications
-import kotlinx.android.synthetic.main.toolbar.*
+import gitlin.kothub.github.api.notifications
 import kotlinx.android.synthetic.main.activity_notifs.*
-import gitlin.kothub.github.api.*
+import kotlinx.android.synthetic.main.toolbar.*
+import org.jetbrains.anko.AnkoLogger
+import org.jetbrains.anko.debug
 
 class NotificationActivity : AppCompatActivity(), AnkoLogger {
```
Comment (fknop) [inline app/src/main/java/gitlin/kothub/ui/NotificationActivity.kt]: Extend the LifecycleAppCompatActivity to create the drawer

File: app/src/main/res/layout/notification_list_view.xml [hunk 2]
```diff
@@ -1,25 +1,58 @@
...
+                android:layout_toEndOf="@id/subjectIcon"
+                android:layout_marginStart="4dp">
+            </TextView>
+
+        </RelativeLayout>
+
+    </LinearLayout>
+
+
+    <View
```
Comment (fknop) [inline app/src/main/res/layout/notification_list_view.xml]: I think you can add dividers with the recycler view API instead.

**Review by fknop — CHANGES_REQUESTED** [review]

File: app/src/main/java/gitlin/kothub/ui/NotificationActivity.kt (hunk 1 above)
Comment (Astalaseven) [reply app/src/main/java/gitlin/kothub/ui/NotificationActivity.kt]: Done

File: app/src/main/java/gitlin/kothub/ui/NotificationActivity.kt [hunk 3]
```diff
@@ -19,9 +20,13 @@ class NotificationActivity : LifecycleAppCompatActivity(), AnkoLogger {
     var notifications: Notifications? = null
         set(value) {
             if (value != null) {
+                val layoutManager = LinearLayoutManager(this)
 
                 notifs.adapter = NotificationAdapter(this, value.notifications)
-                notifs.layoutManager = LinearLayoutManager(this)
+                notifs.layoutManager = layoutManager
+
+                val dividerItemDecoration = DividerItemDecoration(notifs.context, layoutManager.orientation)
```
Comment (fknop) [inline app/src/main/java/gitlin/kothub/ui/NotificationActivity.kt]: Is it necessary to create a val for this ?

**Review by fknop — CHANGES_REQUESTED** [review]
Fix conflicts with master

File: app/src/main/java/gitlin/kothub/ui/NotificationActivity.kt (hunk 3 above)
Comment (Astalaseven) [reply app/src/main/java/gitlin/kothub/ui/NotificationActivity.kt]: I find it cleaner

**Review by fknop — APPROVED** [review]

**Pull request merged**